CONTRIBUTING.md

```diff
@@ -1,8 +1,15 @@
 # Contributing guide for Modern Warfare Cubed
 
-This projects follow [Keep a Changelog] and [Ragnarök Versioning Convention]
+This project follows [Keep a Changelog] and [Ragnarök Versioning Convention]
 
-## Cloning & Setup
+## Writing Good Issues
+
+Submiting a Issues about a bug or feature request is the easiest way to contribute, however, to get the most use out of them, you should follow certain guidelines. 
+
+Be respectful and kind, especially if you're requesting a feature. If you beleve you have found a bug, screenshots/recording are extremely helpful. Check if somone has created a issues for your bug, duplicit reports split up information! If you see a report about a bug you are experiencing, feel free to add more screenshots and steps you have done to find the bug.      
+
+## Editing Code
+This is the hardest, but most helpful, way to contribute, provided you do it correctly    
```

Comment:
No, the contributing guide is ONLY for devs, there is no need to explain good issues as this is the role of issue templates